### PR TITLE
Add tldr extension

### PR DESCRIPTION
**Pull request by pomdtr**
## Description

Browse tldr cheatsheets.

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/17577332/141001974-34020b2d-aaf5-4524-afd3-3ebf8120aaa4.mp4

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I checked that extension's icon has size of `512x512`

### CODE REVIEW

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @pomdtr 👋 

I'm not sure how if I need to do anything in order to get this to work? Both commands are giving the same output.

ELbCA8exSA61LkKA

```
Error: ENOENT: no such file or directory, scandir '/Users/pernielsentikaer/Library/Application Support/com.raycast.macos/extensions/tldr'

readdirSync:node:fs:1390:3
loadPages:index.tsx:12:11
<anonymous>:index.tsx:31:5
invokePassiveEffectCreate:react-reconciler.development.js:12139:24
Object.invokeGuardedCallbackProd:react-reconciler.development.js:9534:14
invokeGuardedCallback:react-reconciler.development.js:9621:35
flushPassiveEffectsImpl:react-reconciler.development.js:12201:13
unstable_runWithPriority:react-reconciler.development.js:716:20
runWithPriority:react-reconciler.development.js:2878:14
flushPassiveEffects:react-reconciler.development.js:12105:18
```

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Comment by pomdtr** [comment]
Thanks, I was assuming that the supportPath existed. It should be fixed now.

File: extensions/tldr/package.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "tldr",
+  "title": "tldr",
+  "author": "pomdtr",
+  "license":"MIT",
+  "description": "Search tldr Pages",
```
Comment (pernielsentikaer) [inline extensions/tldr/package.json:7]:
```suggestion
  "description": "Search tldr Pages - Command Cheatsheet",
```

File: extensions/tldr/package.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  "icon": "tldr.png",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Search TLDR Pages",
```
Comment (pernielsentikaer) [inline extensions/tldr/package.json:12]:
```suggestion
      "title": "Search Pages",
```

Redundant, already added to the subtitle.

File: extensions/tldr/package.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+      "subtitle": "tldr",
+      "description": "Search Commands Cheatsheets.",
+      "mode": "view"
+    },
+    {
+      "name": "refresh-pages",
+      "title": "Refresh TLDR Pages",
```
Comment (pernielsentikaer) [inline extensions/tldr/package.json:19]:
```suggestion
      "title": "Refresh Pages",
```

Redundant, already added to the subtitle — Could also be added as a ActionPanel.item in “Search Pages”

File: extensions/tldr/package.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+      "subtitle": "tldr",
+      "description": "Refresh TLDR Pages",
```
Comment (pernielsentikaer) [inline extensions/tldr/package.json:21]:
```suggestion
      "description": "Refresh tldr Pages",
```

Should be lowercase to match the rest

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @pomdtr 👋 

Awesome new extension, it works a lot better now!

A suggestion could be to add preferences checkboxes where it's possible to deflect platform(s) (not required) — I don't use android/windows, so they're not relevant for me

Above is not a showstopper, it could be added later on also 😄 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

File: extensions/tldr/package.json (hunk 3 above)
Comment (pomdtr) [reply extensions/tldr/package.json:19]: I'll consider it from another PR :)

Comment (pomdtr) [reply extensions/tldr/package.json]: Nevermind, I implemented it

File: extensions/tldr/src/index.tsx [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+      }
+
+      setPlatforms(await readPages());
+    }
+    loadPages();
+  }, []);
+
+  return (
+    <List
+      searchBarPlaceholder="Input command"
```
Comment (pernielsentikaer) [inline extensions/tldr/src/index.tsx:64]: (Search for|Filter by) command... should be used here

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi @pomdtr 💪 

MUCH better! Just a last correction and I think we're ready to go, good job 💯 

Request a new review when you are ready, feel free to contact me here or at [Slack](https://raycastcommunity.slack.com/) if you have any questions.

**Review by pernielsentikaer — APPROVED** [review]
Hi @pomdtr 👋 

Thanks for the corrections, MUCH better! 😄

Let's get this awesome extension shipped 🚢

**Pull request merged**